### PR TITLE
SYS-3237 - Adds new Worflow documentation

### PR DESCRIPTION
**Pull request by hugolesta**
This commit aims to add new Worflow documentation
for every workflow we have in the bestseller-pipeline repository.

### CODE REVIEW

File: docs/build-java-lambda.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Inputs
+
+| Input | Description | Required | Default |
+| :---: | :---: | :---: | :---: |
+| working-dir |  The working dir to work in | `false` | `-` |
+| project_name | The name of the project | `true` | `-` |
+| function-name | The function name | `true` | `-` |
+| java-version | Show file differences that haven't been staged | `false` | `8` |
+| java-distribution | The Java distribution Options: `['temurin', 'zulu','adopt', 'adopt-hostpot', 'adopt-openj9','liberica','microsoft','corretto']` | `false` | `temurin` |
+| build-tool | The build type `['maven', 'gladle']`. Maven by default | `false` | `maven` |
```
Comment (felipem1210) [inline docs/build-java-lambda.md:41]: is gradle, no gladle

File: docs/build.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Build
+
+This workflow will provide a complete solution to build containers for Elastic Container Service.
```
Comment (felipem1210) [inline docs/build.md:3]: I would say "a complete solution to build docker images"

File: docs/deploy-ecs.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+After creating and merging the PR, A GitHub action option will appear to be able to deploy in the chosen environments from the previous example.
+
+## Inputs
+
+| Input | Description | Required |  Default |
+| :---: | :---: | :---: | :---: |
+| project_name | The name of the project that owns the lambdas | `true` | `-` |
+| owner | The Owner's name of the deployment | true` | `-` |
+| environment | The name of the enviroment `['dev','acc','prod','mng']` | `true` | `-` |
+| scheduled-ecs-task | Enable/disable schedul task deployment | `false` | `false` |
```
Comment (felipem1210) [inline docs/deploy-ecs.md:69]: Enable/disable scheduled task deployment

**Review by felipem1210 — CHANGES_REQUESTED** [review]

**Comment by mattialacroce** [comment]
I think documenting each workflow does not add that much value with respect to the time spent. Developer will just copy paste or ask for examples eventually

**Review by mattialacroce — DISMISSED** [review]

**Review by felipem1210 — APPROVED** [review]

**Review by guilhermewolf — APPROVED** [review]

**Pull request merged**